src/app/onprem/license-nag-message-management/nag-message.service.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * CODENVY CONFIDENTIAL
+ * __________________
+ *
+ *  [2015] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+'use strict';
+
+
+/**
+ * This class is handling the service for the nag message
+ * @author Oleksii Orel
+ */
+export class NagMessageService {
+
+  /**
+   * Default constructor that is using resource injection
+   * @ngInject for Dependency injection
+   */
+  constructor($document, cheBodyInjectorSvc, imsLicenseApi) {
+    this.$document = $document;
+    this.cheBodyInjectorSvc = cheBodyInjectorSvc;
+    this.imsLicenseApi = imsLicenseApi;
+
+  }
+
+  /**
+   * Show nag message
+   */
+  showNagMessage() {
+    this.$document.find('body').addClass('license-message-indent');
```

Comment:
NagMessageService is general service (knows nothing about license) - class "license-message-indent"